src/main/java/de/tum/in/www1/jenkins/notifications/SendTestResultsNotificationPostBuildTask.java

```diff
@@ -25,20 +25,16 @@
 import com.cloudbees.plugins.credentials.CredentialsProvider;
 import com.cloudbees.plugins.credentials.common.StandardListBoxModel;
 
-import de.tum.in.ase.parser.ReportParser;
 import de.tum.in.ase.parser.domain.Report;
-import de.tum.in.ase.parser.exception.ParserException;
 import de.tum.in.www1.jenkins.notifications.exception.TestParsingException;
-import de.tum.in.www1.jenkins.notifications.model.*;
+import de.tum.in.www1.jenkins.notifications.model.Commit;
+import de.tum.in.www1.jenkins.notifications.model.TestResults;
+import de.tum.in.www1.jenkins.notifications.model.Testsuite;
 
 import hudson.Extension;
 import hudson.FilePath;
 import hudson.Launcher;
-import hudson.model.AbstractProject;
-import hudson.model.Item;
-import hudson.model.Result;
-import hudson.model.Run;
-import hudson.model.TaskListener;
+import hudson.model.*;
```

Comment:
The static analysis of this repo says that you should have a look at the wildcard imports :)